storage/listkeys.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package storage
+
+// ListKeys returns keys starting from first and upto limit
+func ListKeys(store KeyValueStore, first Key, limit Limit) (Keys, error) {
+	const unlimited = Limit(1 << 31)
+
+	keys := make(Keys, 0, limit)
+	if limit == 0 {
+		// TODO: this shouldn't be probably the case
```

Comment:
yeah, i agree. 0 should probably just mean no keys. if we need a special "unlimited" then -1? but really we shouldn't allow for unlimited behavior

Reply:
I'll change that behavior in a next PR. Would disrupt other packages.